SK.WebApp/src/services/consts.ts

```diff
@@ -0,0 +1,11 @@
+const  Culture = {
+    POLISH: "pl-PL",
+    ENGLISH: "en-US",
+};
+
+const ApiAdresses = {
+    BASE: "https://localhost:44324",
+    LOGIN: "/{culture}/api/User/login",
+};
```

Comment:
`BASE` is not an api address, i.e. endpoint. Also as I've stated earlier, this directory structure needs to go.

My proposition:

```ts
// SK.WebApp/src/api/endpoint.ts

// or even pass some env here like
// process.env.API_BASE_URL
export const baseEndpoint = 'http://localhost:44324';

export const endpoint = {
  login: `${baseEndpoint}/{languageCode}/api/User/login`,
};

export type anyEndpoint = keyof typeof endpoint;
```